packages/utils/number/thousandsDelimiter/thousandsDelimiter.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * This function adds a thousands delimiter(comma) to a numeric string.
+ * @param {string | number} value - Value in numeric format
+ * @throws {Error} If the input value is not a valid number.
+ */
+export function thousandsDelimiter(value: string | number) {
```

Comment:
What do you think about adding an extra argument for what type of delimiter (Ex: ",", "_") the user would like to use?

Reply:
So which do you think is better, receiving the delimiter as a parameter from the user or providing a few options(EX: `,`, `.`, `_`) within the function?

Reply:
I think `delimiter: string = ","` would be fine to give more flexibility.